metrics/web/test/base/presentation/graphs/placeholder_bar_test.dart

```diff
@@ -0,0 +1,131 @@
+// https://github.com/software-platform/monorepo/issues/140
+// ignore_for_file: prefer_const_constructors
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/base/presentation/graphs/colored_bar.dart';
+import 'package:metrics/base/presentation/graphs/placeholder_bar.dart';
+
+void main() {
+  group("PlaceholderBar", () {
+    const _width = 2.0;
+    const color = Colors.grey;
+
+    testWidgets(
+      "can't be created with null width",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(_PlaceholderBarTestbed(width: null));
+
+        expect(tester.takeException(), isAssertionError);
+      },
+    );
+
+    testWidgets(
+      "can't be created with negative width",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(_PlaceholderBarTestbed(width: -2.0));
+
+        expect(tester.takeException(), isAssertionError);
+      },
+    );
+
+    testWidgets(
+      "applies the grey color if no color given",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(_PlaceholderBarTestbed());
+
+        final coloredBar = tester.widget<ColoredBar>(find.byType(ColoredBar));
+
+        expect(coloredBar.color, equals(Colors.grey));
+      },
+    );
+
+    testWidgets(
+      "applies the grey color if the null is passed",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(_PlaceholderBarTestbed());
+
+        final coloredBar = tester.widget<ColoredBar>(find.byType(ColoredBar));
+
+        expect(coloredBar.color, equals(Colors.grey));
+      },
+    );
+
+    testWidgets(
+      "displays the ColoredBar",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(_PlaceholderBarTestbed(width: _width));
+
+        expect(find.byType(ColoredBar), findsOneWidget);
+      },
+    );
+
+    testWidgets(
+      "delegates the given color to the ColoredBar",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(_PlaceholderBarTestbed(
+          color: color,
+        ));
+
+        final coloredBar = tester.widget<ColoredBar>(find.byType(ColoredBar));
+
+        expect(coloredBar.color, equals(color));
+      },
+    );
+
+    testWidgets(
+      "displays the ColoredBar with the given color borders of a 2.0 width",
```

Comment:
should we somehow specify that 2.0 is default width here? maybe just replace 2.0 -> with default width?